src/engine/srv.c

```diff
@@ -423,7 +423,7 @@ dss_srv_handler(void *arg)
 		goto crt_destroy;
 	}
 
-	if (dx->dx_main_xs) {
+	if (dx->dx_main_xs && bio_nvme_configured()) {
```

Comment:
This looks not necessary to me, since the bio_nvme_poll() is just a no-op when NVMe isn't configured.

If you don't create NVMe poll ULT, the scheduler needs be revised a bit, otherwise, it'll report error.